woocommerce/woocommerce-admin
# Refactor panel with withFocusOutside

Fixes #6074 

Refactor panel with `withFocusOutside` and remove `react-click-outside` dependency.

### Screenshots
<img width="329" alt="螢幕快照 2021-01-29 下午3 18 38" src="https://user-images.githubusercontent.com/56378160/106324078-87c50180-6246-11eb-9f23-46548d6061e8.png">

<img width="532" alt="螢幕快照 2021-01-29 下午3 18 46" src="https://user-images.githubusercontent.com/56378160/106324093-901d3c80-6246-11eb-8e97-7f00adb5832a.png">


### Detailed test instructions:
-   Go to WooCommerce home page
-   Click on Display and Help button back and forth, check that the popover and the panel close as expected.
-   Check that the setup store tab continues to work.
